src/components/pages/landing/workshops/WorkshopsList/WorkshopsList.tsx

```diff
@@ -3,18 +3,43 @@ import { Link } from "react-router-dom"
 import s from "./WorkshopsList.module.css"
 
 const WorkshopsList = () => {
+  
   return (
+    <section>
+    <h1 className={s.site_title}>Warsztaty:</h1>
     <ul className={s.workshopsList}>
-        {WORKSHOPS_MOCKS.map(workshop => (
-            <Link to={workshop.id ? `/warsztaty/${workshop.id}` : "/warsztaty"} className={s.workshopLink}>
+        {WORKSHOPS_MOCKS.map(workshop => ( //goes through every workshop listed in workshops_mocks 
             <li>
-                <h1>Nazwa wrsztatu: {workshop.title}</h1> 
-            </li>
-        </Link>
+              <article className={s.card}>
+                <ul>
+                  <img 
+                    //src={workshop.company.logoUrl} //not working unfortunately 
+                    src="https://via.placeholder.com/150"
+                    alt="Workshop" //just in case 
+                    className={s.image}
+                  />
+                  <p className={s.company}>{workshop.company.name}</p>
+                </ul>
+                <div>
+                  <Link to={workshop.id ? `/warsztaty/${workshop.id}` : "/warsztaty"} className={s.workshopLink}>
```

Comment:
Czy tutaj typescript coś krzyczał czy serio jest taka opcja żeby workshop nie miał id?

Reply:
a tu akurat zostawilem co Ty zes napisal. Bo chyba Ty pierwotnie te pliki zrobiles jako przykaldowa. I byly wylistowane sobie warsztaty i po kolei linki. 
Nie zaglebialem sie czemu i jak to dziala zalozylem ze dziala